### DISCUSSION TITLE
[BIG BURN] Gold Idol

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2373] [Ideas]
(                 ,&&&.
             )                .,.&&
            (  (              \=__/
                )             ,'-'.
          (    (  ,,      _.__|/ /|
           ) /\ -((------((_|___/ |
         (  // | (`'      ((  `'--|
       _ -.;_/ \\--._      \\ \-._/.
      (_;-// | \ \-'.\    <_,\_\`--'|
      ( `.__ _  ___,')      <_,-'__,'
       `'(_ )_)(_)_)'

Same as Iron but with gold.

10 weeks burning SFL and Resources

20 supply - Distributed among Top 20 ranked.

(ง •̀_•́)ง ผ(•̀_•́ผ)

Expected burning at least :

```
100.000 SFL (72 SFL Day / Each top player)
100.000 Wood (72 Wood Day / Each top player)
70.000 Stone (50 Stone Day / Each top player)
8.400 Fruits (6 Fruit Day / Each top player)
7.000 Iron ( 5 Iron day / Each top player)
5.600 Gold ( 4 Gold day / Each top player)

```

███╗░░██╗░█████╗░  ██████╗░░█████╗░██╗███╗░░██╗░░░  ███╗░░██╗░█████╗░
████╗░██║██╔══██╗  ██╔══██╗██╔══██╗██║████╗░██║░░░  ████╗░██║██╔══██╗
██╔██╗██║██║░░██║  ██████╔╝███████║██║██╔██╗██║░░░  ██╔██╗██║██║░░██║
██║╚████║██║░░██║  ██╔═══╝░██╔══██║██║██║╚████║██╗  ██║╚████║██║░░██║
██║░╚███║╚█████╔╝  ██║░░░░░██║░░██║██║██║░╚███║╚█║  ██║░╚███║╚█████╔╝
╚═╝░░╚══╝░╚════╝░  ╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░╚══╝░╚╝  ╚═╝░░╚══╝░╚════╝░

░██████╗░░█████╗░██╗███╗░░██╗██╗
██╔════╝░██╔══██╗██║████╗░██║██║
██║░░██╗░███████║██║██╔██╗██║██║
██║░░╚██╗██╔══██║██║██║╚████║╚═╝
╚██████╔╝██║░░██║██║██║░╚███║██╗
░╚═════╝░╚═╝░░╚═╝╚═╝╚═╝░░╚══╝╚═╝   

While big ones burning on Gold Idol, small ones can try other NFTS to burn crops and also use Treasure Island before big one starts.

**Comment by future2back** [comment]
If you think it's easy, we can work with 20 Broken Gold Idol (50% boost) and only 10 available for repair (100% boost) to burn more resources and maybe give use to war items.

**Reply by future2back** [reply]
█▀ █░█ █▄░█ █▀▀ █░░ █▀█ █░█░█ █▀▀ █▀█   █░░ ▄▀█ █▄░█ █▀▄
▄█ █▄█ █░▀█ █▀░ █▄▄ █▄█ ▀▄▀▄▀ ██▄ █▀▄   █▄▄ █▀█ █░▀█ █▄▀


              .--.  .--.
            ._'   |/   :.-,
           |  `'.-;-.;`  .'
           _:._/.'.'.'\.`-.
          /    \.'.'.'/    /
          '-._.;`'-'-';---' 
              /   /|   \
              `--' `-.-'   
                           



We urgently expect a **BIG BURN**, from smallest to the biggest.

In times of tidying up our homes.

Meanwhile the dev team gets 15 weeks to work on new gameplay like:

- PVP
- PVE
- Co-op 
- Guild Wars
- Professions

The game needs to get out of this sameness asap. Game is boring, players need entertainment!
We have to stop generating resources like crazy, this is not entertaining, and start to burn.  Devs side have to stop creating worthless NFT so as not to pollute our system.
The evolution of the game takes place at a very slow pace, which is very harmful.
With an open world we have an infinity of possibilities to be explored. How much longer will we have to wait? Apart from the other things we still have to work on, like animals and magic beans. Come on guys!? Time is flying...
Isn't it time to hire? Why not partner with an online tech school and bring in new developers who want to learn and contribute to the project? This would greatly enrich the project in addition to helping with dissemination.  There are a lot of people wanting to learn and looking for a real experience, I believe this project would be perfect for that. Current programmers/artists can be promoted to project leads and supervise interns at bootcamps, making it faster to complete tasks. We need artists and developers helping us. We have to think big to be big.






           /\
          |==|
          |  |
          |  |
          |  |
         /____\
         |S  S|
         |F  F|
         |L  L|
         |    |
        /| |  |\
       / | |  | \
      /__|_|__|__\
         /_\/_\
         ######
        ########
         ######
          ####
          ####
           ##
           ##  
           ##
           ##

**Comment by LandLadyFarmer** [comment]
This is just stupid and the worst thing ever read, stop using github as your personal complaining board and start propose valid things in depth and do a lot of ground work yourself before proposing...

A lot of things proposed here lately on github is mostly a load of crap and not any good intresting proposals or suggestions not even worth coment on any more
it's only dcol, super synapse and maybe some others in latest time that leaves thorough, well-thought-out proposal where the groundwork is partially done and elaborated that is was devs need not a load of crap proposals all over github

Why don't you start to investigate and contact all those that you mention and come with a valid proposal, they are 4 devs so how do you think they can do all the things you and others suggest????
If we want big things we have to do big things so instead of suggesting and complain about stupid things  here start doing the work so it can be easy understand and with work done for devs to implement or start to think about...
Start to hire? what? have you looked at the econamy so you know how it looks to make a suggestion like that or it's based on discord talk that devs make so much money from syncs???  You get what you pay for.....

econamy is pretty stable, no massiv hording, burn is good so for now it's stable so let devs implement what they are doing right now and get it stabliced so we can go forward so stop using github as your playground and if you want things fast do the work yourself..

**Reply by future2back** [reply]
Stupid is your comment. You seem to be very active here and must have contributed a lot to this project right?  What are your ideas?
